app/main.go

```diff
@@ -136,8 +137,13 @@ func main() {
 		SyncPeriod:               *syncPeriod,
 	})
 
-	h := web.NewHandler(config)
+	metrics, err := CreateMetrics(context.Background(), config)
+	if err != nil {
+		log.Fatalf("failed creating metric dependencies: %v", err)
```

Comment:
Nit: this might be a tad confusing since this method doesn't really make any operations on metrics themselves rather just initializes adaptor/collector for them

Reply:
Updated the log msg here.